_templates/exchanges.html

```diff
@@ -286,6 +286,8 @@ <h3 id="argentina" class="no_toc">Argentina</h3>
           <a class="marketplace-link" href="https://www.buenbit.com/">Buenbit</a>
           <br>
           <a class="marketplace-link" href="https://www.satoshitango.com/">SatoshiTango</a>
+          <br>
+          <a class="marketplace-link" href="https://bitex.la/">Bitex</a>
```

Comment:
Exchanges are listed alphabetically under each category/country. Please move this to ensure that is the case with this addition.